### PR TITLE
backend: Bootstrap Express server

### LINKED ISSUE #2: Bootstrap backend server
Initialize the Node.js server:
- Create a Node package by doing `yarn init -y` or `npm init -y` in the `backend` directory.
- Add dependencies like `express`
- Setup a basic Express server with a `/health` endpoint, i.e adding a simple route like

```js
app.get('/health', function (req, res, next) {
  res.json({status: 'UP'});
});
```

### PR DESCRIPTION
**Pull request by Tan-2029**
Fixes #2

### CODE REVIEW

File: backend/index.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const { Router } = require('express');
```
Comment (onprem) [inline backend/index.js]: Let's move this file inside a directory named `src`. This way, the `backend/src` directory will contain all of the code.

File: backend/package.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "backend",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (onprem) [inline backend/package.json:8]: Just a suggestion, adding a script called `start` is the idiomatic/preferred way in Node to start the application. Add sone like,
```json
    "start": "node src/index.js"
```

After this, you can simply run `npm run start` or `yarn start` to run the server.

**Review by onprem — COMMENTED** [review]
Just a few nits, otherwise LGTM!

**Comment by onprem** [comment]
IMO, a better title would be `backend: Bootstrap Express server` and then you can mention somewhere in the description that `Fixes #2`.

File: backend/index.js (hunk 1 above)
Comment (Tan-2029) [reply backend/index.js]: Done sir

File: backend/package.json (hunk 2 above)
Comment (Tan-2029) [reply backend/package.json:8]: On it sir

Comment (Tan-2029) [reply backend/package.json:8]: Sir, I had a doubt regarding port. I've heard that ports above 3000 are generally used, while the port 3000 is itself reserved for React. Is there any such significance or good practice to use a certain port number for certain things?

Comment (onprem) [reply backend/package.json:8]: There isn't. You can use any port that you find easy to remember. As with CRA, the default one is `3000`, we avoid using that in backend code.

**Review by onprem — APPROVED** [review]
LGTM

**Pull request merged**